### ISSUE TITLE
(feature request) full black theme as well as dark theme

### ISSUE THREAD

**Issue by realeatham** [tfonteyn/NeverTooManyBooks#49]
See title

**Comment by tfonteyn** [comment]
Not sure what you mean?

The app offers (in Settings) to use Light/Dark or "follow-the-device day/night". If you as I suspect mean extra Themes to those, then I regret to say, but I currently have no plans. There are other things which will get priority.

**Comment by realeatham** [comment]
Yes, I meant another theme. Can you leave the issue open incase you or someone else wants to add it in the future?

**Comment by tfonteyn** [comment]
sure - but it will require some work, it's not just adding it to a list. Day/Night is something Android supports via an API. Extra Themes work slightly differently.

**Comment by tfonteyn** [comment]
@realeatham 

Here's your chance - I implemented #46 in such a way that I can now support multiple themes.

My classic "Blue/Grey" scheme is defined in style name="Theme.App", in this [source file](https://github.com/tfonteyn/NeverTooManyBooks/blob/dev/app/src/main/res/values/themes.xml)

I don't know how familiar (if at all) you are with Android development or its color use?
If you want to have a go with the [Theme Builder](https://material-foundation.github.io/material-theme-builder/) perhaps?

And no worry, I don't expect a ready-solution; but if you can play around with that tool (or any other of your liking) and come up with color definitions for the "primary" and "secondary" colors (see above source code), there are 9 colors + 1 for the background to define. The "surface" colors I can extrapolate from those prim/sec colors.

**Comment by ChristianGruen** [comment]
Personally, I think that the current light/dark/system choice is completely sufficient.

One thing I noticed: When I choose »Dynamic Colors«, the top area gets completely black (with my current Android settings, at least):

<img src='https://github.com/user-attachments/assets/aceabe0e-ecbf-4a77-9623-11f1c33c6e3e' width='200'/>

The Blue-gray color scheme looks nice all the time:

<img src='https://github.com/user-attachments/assets/63affe4f-1610-4705-adc7-6c0b3c53cca2' width='200'/>

**Comment by tfonteyn** [comment]
on "Dynamic Colors":  as the saying goes.... Android moves in mysterious ways...

**Comment by tfonteyn** [comment]
closing... anyone reading this and want to contribute, I'll certainly help.
